cmd/m3/admin-add.go

```diff
@@ -67,20 +67,24 @@ func adminAdd(ctx *cli.Context) error {
 	}
 
 	// perform the action
-
-	apptCtx, err := cluster.NewEmptyContext()
+	cnxs, err := GetGRPCChannel()
 	if err != nil {
 		fmt.Println(err)
 		return err
 	}
+	defer cnxs.Conn.Close()
+	// perform RPC
+	_, err = cnxs.Client.AdminAdd(cnxs.Context, &pb.AdminAddRequest{
+		Name:  name,
+		Email: email,
+	})
 
-	admin, err := cluster.AddAdminAction(apptCtx, name, email)
 	if err != nil {
-		fmt.Println("Error adding user:", err.Error())
-		return err
+		fmt.Println(err)
+		return nil
```

Comment:
error handling?

Reply:
This will print the error for the users to understand why the request failed